services/wikiapiary/wikiapiary-extension.service.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+const Joi = require('joi')
+const { metric } = require('../text-formatters')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  query: Joi.object({
+    results: Joi.object()
+      .required()
+      .pattern(/^.+:.+$/, {
+        printouts: Joi.object({
+          'Has website count': Joi.array()
+            .required()
+            .items(Joi.number().required()),
+        }).required(),
+      }),
+  }).required(),
+}).required()
+
+/**
+ * This badge displays the total usages of a MediaWiki extensions via
+ * Wikiapiary.
+ *
+ * {@link https://www.mediawiki.org/wiki/Manual:Extensions MediaWiki Extensions Manual}
+ */
+module.exports = class WikiapiaryExtension extends BaseJsonService {
+  static category = 'downloads'
+  static route = {
+    base: 'wikiapiary/extensions',
+    pattern: ':type/:extension',
```

Comment:
> Technically, this works for "Farms", "Generators", and "Hosts" as well, so it could be worth renaming this?

Yeah lets drop the terms "extension" and "downloads". It seems like what we're actually measuring here is more like install base rather than downloads? (and in principle there could be other metrics). How about making the URLs:

`wikiapiary/:variant(extension|skin|farm|generator|host)/installs/:name`

https://github.com/badges/shields/blob/master/doc/badge-urls.md

Wordpress plugins/themes might be a good reference point for the terminology.